Cargo.toml

```diff
@@ -64,6 +64,8 @@ zbus = { version = "5.0", default-features = false, features = ["url"] }
 [dev-dependencies]
 serde_json = "1.0"
 reis = { version = "0.4", features = ["tokio"] }
+pipewire = "0.8.0"
+tokio = { version = "1.41", features = [ "rt-multi-thread", "macros" ] }
```

Comment:
can't we use async-std instead? It's already a dependency.

Reply:
Yeah, even in my original example- I had used async_std but I saw this crate is mostly based on tokio so tokio made more sense. Plus many users will majorly be based on tokio only improving the copy/pasting ability.